collection-scripts/gather_sos_networking

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="/must-gather"
+LOG_PATH="${BASE_COLLECTION_PATH}/sos_info/networking"
+
+# use quorum-guard becuase it is privileged and uses hostNetwork
+QUORUM_GUARD_DATA=($(oc get pods -n openshift-etcd -l name=etcd-quorum-guard -o jsonpath='{range .items[*]}{.metadata.name}{","}{.status.hostIP}{" "}{end}')) || {
+  echo "ERROR: No networking data was collected"
+  exit 0
+}
+
+if [[ -z $QUORUM_GUARD_DATA ]];then
+  echo "ERROR: No data was returned from oc request"
+  exit 0
+fi
+
+set -o nounset
+
+COMMANDS=(
+  ip_-4_rule
+  ip_-6_rule
+  ip_maddr_show
+  ip_neigh_show_nud_noarp
+  ip_-o_addr
+  ip_-s_-d_link
+  ip_-6_route_show_table_all
+  ip_-d_address
+  ip_mroute_show
+  ip_netns
+  ip_route_show_table_all
+  ip_-s_-s_neigh_show
+)
+
+run_command(){
+  local pod="$1"
+  local node="$2"
+  local command="$3"
+  eval oc exec  -n openshift-etcd $pod -- /bin/sh -c '"${command//_/ }"' 1> "$LOG_PATH/${node}-${command}.log" || true
+}
+
+mkdir -p ${LOG_PATH}
+echo "INFO: Starting getting networking information"
+
+for pod in "${QUORUM_GUARD_DATA[@]}"; do
+  # create array element [0] is pod name [1] is node ip
+  IFS=',' read -a data <<< "$pod"
+  echo "Getting networking data from hostNetwork on \"${data[1]}\" via \"${data[0]}\""
+  for command in "${COMMANDS[@]}"; do
+    run_command ${data[0]} ${data[1]} $command &
```

Comment:
If the commands are defined with spaces and double-quotes, this needs to be quoted:
```
run_command ${data[0]} ${data[1]} "$command"
```